src/utils/rootkeypackage_utils/src/rootkeypackage_parse.c

```diff
@@ -0,0 +1,827 @@
+
+#include "aduc/rootkeypackage_parse.h"
+#include <aduc/logging.h>
+#include <aduc/string_c_utils.h> // for IsNullOrEmpty
+#include <azure_c_shared_utility/strings.h>
+#include <azure_c_shared_utility/vector.h>
+#include <base64_utils.h> // for Base64URLDecode
+
+EXTERN_C_BEGIN
+
+/**
+ * @brief Frees resources for an ADUC_RootKey object.
+ *
+ * @param node The pointer to the root key object.
+ */
+void ADUC_RootKey_Free(ADUC_RootKey* node)
```

Comment:
Free or DeInit. I have to call a free(node) before this is free meaning we're de-initializing the contents.

Reply:
Made this and hash _Free -> _DeInit